romancal/scripts/make_regtestdata.sh

```diff
@@ -36,7 +53,7 @@ do
     cp ${fn}_*.asdf $outdir/roman-pipeline/dev/truth/WFI/$dirname/
 
     # uncal file doesn't belong in truth.
-    rm $outdir/roman-pipeline/dev/truth/WFI/image/${fn}_uncal.asdf
+    rm $outdir/roman-pipeline/dev/truth/WFI/${dirname}/${fn}_uncal.asdf
```

Comment:
Minor bugfix. The loop processing the `grism` tries to delete the `image` file (and the `grism` uncal isn't deleted).